lib/validator.js

```diff
@@ -330,6 +330,13 @@ validator.validatePutRequest = function(req, schema) {
         throw new Error('Invalid query. No schema for ' + req.table);
     }
 
+    var missingColumn = Object.keys(req.attributes).find(function(attrName) {
```

Comment:
This will be dealt with differently in the Cassandra module, so retracting my question.